smoke-tests-otel-starter/spring-boot-3/src/main/resources/application-jdbc-driver-config.properties

```diff
@@ -1,4 +0,0 @@
-spring.datasource.username=root
-spring.datasource.password=root
-spring.datasource.url=jdbc:otel:h2:mem:db
```

Comment:
where is the URL set - if not here?

Reply:
If not defined, Spring Boot provides a default configured H2 datasource.